src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Admin/Controllers/RequestsController.cs

```diff
@@ -29,7 +31,7 @@ public RequestsController(IFeedbackService feedbackService)
         /// <returns>Action result</returns>
         public ActionResult Index()
         {
-            var feedbacks = _feedbackService.Get().ConvertAll(f => new RequestsViewModel(f));
+            var feedbacks = ((List<Feedback>)_feedbackService.Get()).ConvertAll(f => new RequestsViewModel(f));
```

Comment:
cast?

Reply:
this cast is necessary

Reply:
explain why

Reply:
_feedbackService.Get() returns ICollection or IEnumerable and in Get() we return _getAllFeedbacksQuery wtich is from IQuery<ICollection<Feedback>, GetAllCriteria>

Reply:
There is a way to do same thing without cast

Reply:
using *as* keyword? or what?

Reply:
please tell me how to do it, mb some useful link or something